### PR TITLE
3.1.0 Merging

### PR DESCRIPTION
**Pull request by zumbak04**
v3.1.0 of CK2 merged into the mod.

Changelog:
- Cathedral is available for Light and Tidemother followers.
- Great Temple is available for shamanic (except Tidemother) and druidic followers.
- Grand Amphitheater is available for trolls and ogres.
- Grand University is available for non-tribals and non-nomads.
- Grand Library is available for non-tribals and non-nomads.
- Pyramid is avaliable for tol'vir.
- Stonehenge is avaliable for shamanic and druidic followers.
- ~~Aztec~~ Step Pyramid is avaliable for trolls and Loa worshipers.

**Note for Testers:** please, test new features from v3.1.0. They all should work without problems.

### CODE REVIEW

**Comment by zumbak04** [comment]
> Works fine, also a note- if you change religion(like fel) as you build a great cathedral, it stops building.

As it should...
Actually, we must ban all wonders for evil religions and add Necropolis, Demon Portals, Twilight Citadels and so on.

File: common/wonders/00_wonders.txt [hunk 1]
```diff
@@ -804,11 +804,18 @@ wonder_statue_ruler = {
 	picture = GFX_wonder_strip_ruler_statue
 	model = wonder_statue
 	potential = {
+		# Warcraft
```
Comment (DukeJeans) [inline common/wonders/00_wonders.txt]: Why are great statues banned for evil religions? Surely Demon/Old God minions have erected monuments in honor of themselves?

Comment (zumbak04) [reply common/wonders/00_wonders.txt]: Statues are fine, but there's a lot of upgrades that don't look good with evil guys. Like tavern, statues of family (even if you don't have it), hypocausts, heating pipes.

**Comment by MemeWizard** [comment]
No loc

<details>
<summary>Click to expand</summary>

ck2_5

</details>

This tech don't gives you -cost +speed to wonder's construction
<details>
<summary>Click to expand</summary>

ck2_8

</details>

>Zunist opinion

<details>
<summary>Click to expand</summary>

ck2_9

</details>

Why you can build fountain of youth if you can't get mystic?
Why you can build herb garden if you can't join hermetics?

<details>
<summary>Click to expand</summary>

ck2_13
ck2_14


</details>

>cross
>biblical scene

<details>
<summary>Click to expand</summary>

ck2_19
ck2_20


</details>

>Vitruvius
>Plato

<details>
<summary>Click to expand</summary>

ck2_21
ck2_22

</details>

**Comment by MemeWizard** [comment]
Also this
<details>
<summary>Click to expand</summary>

ck2_1

</details>

**Comment by zumbak04** [comment]
@MemeWizard, implemented your things except wonder_aztec_pyramid localization because it's vanilla bug and I want Paradox to fix it :3

**Comment by MemeWizard** [comment]
Great works have models on the map in vanilla, but do not have models on the map in the mod

**Comment by zumbak04** [comment]
> Great works have models on the map in vanilla, but do not have models on the map in the mod

It's almost impossible to place the models on the map, because the Nudge tool (thing that helps with model placement) don't show the great works. Waiting for Paradox to do something with it.

**Comment by MemeWizard** [comment]
Strategic Exclaves don't upgrade walls
<details>
<summary>Click to expand</summary>

ck2_17
ck2_18
ck2_19

</details>

**Comment by MemeWizard** [comment]
Great works have no background for unique landscape from the modification.

Scorched earth uses a background from plains, forest and farmlands.

<details>
<summary>Click to expand</summary>

ck2_26
ck2_27
ck2_28

</details>

Marsh uses a background from jungle.

<details>
<summary>Click to expand</summary>

ck2_25

</details>

**Comment by MemeWizard** [comment]
This button does not exist in the master branch on the screen of the Old Gods and should not exist in this branch either.

<details>
<summary>Click to expand</summary>

ck2_31

</details>

**Comment by zumbak04** [comment]
> Strategic Exclaves don't upgrade walls

Now they give Fort Level instead.

> Great works have no background for unique landscape from the modification.
> 
> Scorched earth uses a background from plains, forest and farmlands.

Most likely, it's hardcoded. Couldn't find lines that define background depending on terrain.

> Marsh uses a background from jungle.

Marshes use jungle background in vanilla as well

> This button does not exist in the master branch on the screen of the Old Gods and should not exist in this branch either.

Commented out again.

**Comment by MemeWizard** [comment]
No unique loc for description of the mod terrain
<details>
<summary>Click to expand</summary>

image

</details>

How it's should be:

<details>
<summary>Click to expand</summary>

image


</details>

**Comment by zumbak04** [comment]
> No unique loc for description of the mod terrain

Added.

**Pull request merged**